Modules/Office365DSC/Modules/O365DSCReverse.psm1

```diff
@@ -163,6 +163,28 @@ function Start-O365ConfigurationExtract
                     $ComponentsToExtract.Contains("chck" + $resourceName)) -or
                 $AllComponents -or ($null -ne $Workloads -and $Workloads.Contains($currentWorkload)))
             {
+                $shouldSkip = $false
+                $usedPlatforms = Get-ResourcePlatformUsage -Resource $resourceName -ResourceModuleFilePath $ResourceModule.FullName
+
+                foreach($platform in $usedPlatforms)
+                {
+                    # we will skip PnP if there was a problem connecting to a specific site
+                    # it could be a permissions issue
+                    # if it was a problem with connecting to the admin site, then we know that all else will fail as well so no need to continue
+                    if($platform -eq 'PnP' -and $null -ne $Global:SPOAdminUrl -and $Global:SPOConnectionUrl -ne $Global:SPOAdminUrl)
+                    {
+                        continue
+                    }
+                    $isAvailable = Check-PlatformAvailability -Platform $platform
+                    $shouldSkip = $shouldSkip -or !$isAvailable
+                }
+
+                if($shouldSkip)
+                {
+                    Write-Verbose "Skipped [$resourceName] because of connection problems with the used MsCloudLogin platform"
+                    continue;
```

Comment:
Should we show this information to the user? Maybe Write-Information or Write-Warning to show it in the UI instead of just skipping it?

Reply:
not really a fan of that option. this would mean that all the current errors that are shown would just be replaced with this. still a lot of noise. What i was thinking but is currently not implemented is to show some info for only the first failure. Ie:
```
ExchangeOnline connection failed, all dependant resource extraction will be skipped
```

Reply:
Okay, sounds good